java/google/registry/model/reporting/HistoryEntry.java

```diff
@@ -63,6 +63,10 @@
     HOST_PENDING_DELETE,
     HOST_UPDATE,
     /**
+     * Resource was created by an escrow file import.
+     */
+    RDE_IMPORT,
```

Comment:
I don't exactly agree with how this is intended to be used.  Currently, domains, hosts, and contacts always have an initial DOMAIN_CREATE, HOST_CREATE, and CONTACT_CREATE history entry, respectively.  This is an invariant of the system, the guaranteed presence of said instances have proven useful in the past to us for some fixes and data migrations, and it seems reasonable to reuse these existing history entry types for resources imported via RDE.

I could see an argument being made that we don't actually know how to create the correct initial creation history entries for RDE-imported resources, and that what we're really storing instead is a history entry at the time of transfer across registries.  I don't know if that's a distinction that needs to be made, or if it's fine for the creation history entries to implicitly mean that's when they were created in the current registry, not necessarily that they didn't previously exist in someone else's authoritative system.

I think my preferred solution would be to use the existing CREATE history entry types, and then setting a message along the lines of "Created from RDE import" as the reason on the history entry.  That way we preserve the existing invariant of these initial creation history entry types always existing for all resources, while still being able to find out which ones actually represent a transfer from another registry (if necessary).